specifications/consensus/consensus.md

```diff
@@ -1114,6 +1114,8 @@ fn process_proposal_msg(&mut self, proposal_msg: ProposalMsg) -> Result<()> {
     let ProposalMsg {proposal, sync_info} = proposal_msg;
     self.ensure_epoch(proposal.epoch())?;
     self.ensure_round(proposal.round(), sync_info)?;
+    proposal_msg.verify()?;
```

Comment:
in reality we check the epoch, then we verify using `validator_verifier`, then we check the round/sync_info